dlt/cli/ai_command.py

```diff
@@ -0,0 +1,59 @@
+import shutil
+from pathlib import Path
+from typing import get_args, Literal, Set
+
+import dlt
+from dlt.cli import echo as fmt
+from dlt.common.runtime import run_context
+from dlt.common.storages.file_storage import FileStorage
+
+
+TSupportedIde = Literal[
+    "cursor",
+    "continue",
+    "cline",
+    "claude_desktop",
+]
+
+SUPPORTED_IDES: Set[TSupportedIde] = set(get_args(TSupportedIde))
+IDE_RULES_SPECS: dict[TSupportedIde, tuple[str, str]] = {
+    "cursor": ("cursor", ".cursorrules"),
+    "continue": ("continue", ".continue"),
+    "cline": ("cline", ".clinerules"),
+    "claude_desktop": ("claude", ".claude"),
+}
+RULES_TEMPLATE_RELATIVE_PATH = "sources/_ai_static_files/rules"
+
+# TODO Claude Desktop: rules need to be named `CLAUDE.md`, allow command to append to it
+# TODO Continue: rules need to be in YAML file, allow command to properly edit it
+# TODO generate more files based on the specifics of the source README and the destination
+
+def ai_setup_command(ide: TSupportedIde = None) -> None:
+    """Get AI rules files into your local project for the selected IDE.
+    
+    Get the source and destination directories for the rules files.
+    Files found in the source directory will be copied into the destination directory.
+    """
+    specs = IDE_RULES_SPECS[ide]
+
+    src_storage = FileStorage(str(Path(dlt.__file__).parent.joinpath(RULES_TEMPLATE_RELATIVE_PATH)))
```

Comment:
I'd go for something way dumber. you have
- /ai folder
- it has subfolders with names of the editors
- those subfolders contains files to copy into `run_dir`

the command looks if it has a subfolder with editor name and copies all the files

this way you can add and remove editors, add more files etc. without changes in the code. WDYT?

OFC you should not overwrite existing files.